src/main/kotlin/frc/robot/subsystems/conveyor/Conveyor.kt

```diff
@@ -56,7 +64,15 @@ class Conveyor private constructor(private val io: ConveyorIO) : SubsystemBase()
     }
 
     override fun periodic() {
+        LoggedTunableNumber.ifChanged(
+            hashCode(), { kPIDSVA: DoubleArray ->
+                io.setPID(
+                    kPIDSVA[0], kPIDSVA[1], kPIDSVA[2], kPIDSVA[3], kPIDSVA[4], kPIDSVA[5]
+                )
+            }, kP, kI, kD, kS, kV, kA
+        )
```

Comment:
https://chatgpt.com/share/ff726f5a-d6df-45bf-9460-b525e9f2d2d2
You can make this much more concise by using a lambda (IntelliJ IDEA possibly suggests this) and possibly `vararg`

Reply:
I wouldn't say this is more concise; I think it's actually less readable because your example omits the kPIDSVA array. Without it, it’s harder to understand what each part of the array represents.